docs/PullRequests.md

```diff
@@ -0,0 +1,29 @@
+# Pull Requests
+
+## Before opening a pull request
+
+Before opening a pull request, please be sure that:
+
+- your code follows our [contributing guidelines](./ContributionGuidelines.md) and that your [code quality checks](./CodeQuality.md) pass locally.
+- you tested your code on an ios and android and made sure that both are consistent with our visual guidelines.
```

Comment:
Is there a link to our visual guidelines?

Reply:
adicionei um link pro zero height ;) 
done in c5e361b